src/algebra/lie/from_cartan_matrix.lean

```diff
@@ -0,0 +1,146 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import algebra.lie.free
+import algebra.lie.quotient
+
+/-!
+# Lie algebras from Cartan matrices
+
+Split semi-simple Lie algebras are uniquely determined by their Cartan matrix. Indeed, if `A` is
```

Comment:
Sure. I was planning to introduce a formal definition for this and then reference that but since I haven't (yet) I'll add the informal definition here.